src/mock-connector/cart.js

```diff
@@ -0,0 +1,39 @@
+import createProduct from './utils/createProduct'
+import withAmpFormParser from '../middlewares/withAmpFormParser'
+
+const cart = (req, res) => {
+  console.log('method', req.method)
```

Comment:
Should remove these console logs, here and elsewhere.

Reply:
I removed this one and the one below it that output the body, but I left the params because the mock cart method doesn't really do anything else, so these might be helpful for a developer learning about connectors.